RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -38,6 +38,16 @@ import RevenueCat
     private(set) var appIsLatestVersion: Bool = defaultAppIsLatestVersion
     private(set) var purchasesProvider: CustomerCenterPurchasesType
 
+    var shouldShowAppUpdateWarning: Bool {
+        return !appIsLatestVersion && (configuration?.support.shouldWarnCustomerToUpdate ?? false)
+    }
+
+    /// Whether or not the user needs to update their app version to contact support.
+    var appUpdateRequiredToContactSupport: Bool {
```

Comment:
we don't entirely _require_ but rather _suggest_, right?

Reply:
With this implementation, I built it the way the copy for the checkbox on the dashboard is worded and made it _required_ to contact support in the WrongPlatformView & the Restore Purchases flow.

The checkbox's copy reads:
> Require users to update to the latest version of your app before contacting support

That said, I don't think it's great to prevent people from contacting support and would suggest that we instead show a message or button in the alert urging the user to update in addition to letting them contact support.